sunpy/instr/fermi.py

```diff
@@ -459,39 +459,32 @@ def met_to_utc(timeinsec):
 
     Returns
     -------
-    `datetime.datetime`
-        The input Fermi Mission Elapsed Time converted to a datetime object.
+    `astropy.time.Time`
+        The input Fermi Mission Elapsed Time converted to a Time object.
```

Comment:
same as above

Reply:
But it's already mentioned once, right? Should we do it again?